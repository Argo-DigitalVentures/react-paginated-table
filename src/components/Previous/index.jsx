import React from 'react';

function Previous(props) {
  return (
    <button {...props}>Previous</button>
  );
}

export default Previous;
