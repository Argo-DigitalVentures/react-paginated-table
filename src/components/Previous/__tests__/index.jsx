import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Previous from '../';

describe('<Previous />', () => {
  it('should accept className, style, and other DOM props', () => {
    const tree = shallow(
      <Previous
        className="test"
        style={{
          color: 'green',
        }}
        aria-label="previous page button"
      />,
    );
    expect(toJson(tree)).toMatchSnapshot();
  });
});
