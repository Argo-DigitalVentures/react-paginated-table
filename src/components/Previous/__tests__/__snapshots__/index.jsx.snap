exports[`<Previous /> should accept className, style, and other DOM props 1`] = `
<button
  aria-label="previous page button"
  className="test"
  style={
    Object {
      "color": "green",
    }
  }>
  Previous
</button>
`;
